docs/deployment/uninstallation.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Uninstallation"
+layout: default
+sort: 3
+---
+
+# Uninstallation
+{: .no_toc}
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+---
+
+## Uninstallation
+
+If you followed the deployment instructions above you can simply do:
+
+```bash
+kubectl -n nfd delete NodeFeatureDiscovery my-nfd-deployment
+```
+
+Optionally, you can also remove the namespace:
+
+```bash
+kubectl delete ns nfd
+```
+
+See the [node-feature-discovery-operator][nfd-operator] and [OLM][OLM] project
+documentation for instructions for uninstalling the operator and operator
+lifecycle manager, respectively.
+
+<!-- Links -->
+[nfd-operator]: https://github.com/kubernetes-sigs/node-feature-discovery-operator
+[OLM]: https://github.com/operator-framework/operator-lifecycle-manager
+
+
+## Removing feature labels
+
+From the Operand repository NFD-Master has a special `-prune` command line flag
+for removing all nfd-related node labels, annotations and extended resources 
+from the cluster.
```

Comment:
Just a sidenote, could this be done automatically with the operator, e.g. with finalizer or smth, WDYT? We could open a separate issue about that

Reply:
yes, but needs to be optional, like a entry in the `NodeFeatureDiscoveries` CRD, that is set false by default

Reply:
https://github.com/kubernetes-sigs/node-feature-discovery-operator/issues/187